content/posts/March-2020.md

```diff
@@ -38,3 +38,31 @@ draft: false
 [(#458)]: https://github.com/packit-service/packit-service/pull/458
 [(#455)]: https://github.com/packit-service/packit-service/pull/455
 
+
+## Week 11 (March 9th - March 13th)
+
+
+
+### packit
+* Log messages when loading a package config are improved to be more clear [(#755)]
+* We have changed how packit treats version and release in the spec file [(#748)]
+    * We handle the `git-describe` output better which should help when tags contain dashes [(#759)]
+* Packit does not add the patches inside the macro now [(#760)]
+
+
+### packit-service
+* Comments from packit-service about successful `propose-update` in the issue which triggered the update now include the URL of 
+  the new pull-request in Fedora [(#472)]
```

Comment:
had to read this one like 3 times - what a complex sentence :D let's leave it as it is